src/upgrades/mod.rs

```diff
@@ -769,6 +777,29 @@ pub fn prepare_upgrade_data_migrations(
                 pre_0_14_0::UpgradeAggregateStoreTrustAnchorProxy::upgrade(TA_PROXY_SERVER_NS, mode, config)?;
 
                 Ok(Some(UpgradeReport::new(aspa_configs, true, versions)))
+            } else if versions.from < KrillVersion::release(0, 14, 3) {
+                // Check all possibly affected stores for corrupted files resulting
+                // from issue #1160 and fix them if needed.
+                let mut migrated = false;
+
+                pre_0_14_3::upgrade_tasks(config)?;
+                pre_0_14_3::upgrade_status(config)?;
+
+                migrated = pre_0_14_3::upgrade_agg::<CertAuth>(CASERVER_NS, config)? || migrated;
```

Comment:
Could these `migrated = N || migrated` lines not be done as `migrated |= N` ?

Reply:
The left-hand function must be evaluated first, so I don't think so.